src/scripts/main.js

```diff
@@ -1,29 +1,65 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  // Creating HTML-element of notofication, adding classes and child elements
+  document.body.appendChild(notification);
+
+  notification.classList.add('notification', type);
+
+  const notificationHeader = document.createElement('h2');
+
+  notificationHeader.textContent = title;
+  notificationHeader.className = 'title';
+  notification.appendChild(notificationHeader);
+
+  const notificationMessage = document.createElement('p');
+
+  notificationMessage.textContent = description;
+
+  notification.appendChild(notificationMessage);
+
+  // Positioning of the notification element
+  notification.style.position = 'absolute';
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  // Timeout for notificatoon disappearing
```

Comment:
There is a typo in the comment: 'notificatoon' should be corrected to 'notification'.